### ISSUE TITLE
Colours inverted while streaming to steamlink (Desktop OpenGL NV12).

### ISSUE THREAD

**Issue by RlndVt** [ValveSoftware/steam-for-linux#5485]
#### Your system information

* Steam client version (build number or date): April 2 2018
* Distribution (e.g. Ubuntu):  Ubuntu 18.04
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

GPU: AMD Rx480

#### Please describe your issue in as much detail as possible:
Colours get inverted (blue->red) when streaming to Steamlink. When a game is started (e.g. Rocket League) the colours are still inverted during loading but it stops when the actual game is loaded. 

In other words, when the encoder is 'Desktop OpenGL NV12 + ...' the colours are inverted but with 'Game Delayed OpenGL NV12 + ...' it is fine.

#### Steps for reproducing this issue:

1. Open Steam
2. (Optional) Start Big Picture
3. Connect to PC from steam link.

**Comment by kisak-valve** [comment]
Hello @RlndVt, please take a look at #5434 and see if that's the issue you're seeing.

**Comment by RlndVt** [comment]
Yes it's the same. I haven't tried the workaround but have faith that it will work.

Thanks
